src/main/java/seedu/trackermon/logic/parser/SortCommandParser.java

```diff
@@ -27,15 +27,21 @@
  */
 public class SortCommandParser implements Parser<SortCommand> {
 
+    protected static final String VALUE_ASC = "ASC";
+    protected static final String VALUE_DSC = "DSC";
```

Comment:
Just curious, is there a need for VALUE_ASC and VALUE_DSC to be protected?

Reply:
I used it in parserUtil, so can't make it private!

Reply:
Noted